server/app.js

```diff
@@ -1,9 +1,13 @@
 const express = require('express');
-
+const compression = require('compression');
+const bodyParser = require('body-parser');
 const controllers = require('./controllers/routes');
 
 const app = express();
 
+app.disable('x-powered-by');
+app.use(bodyParser.json());
```

Comment:
"you don't need to use body parser. Instead, you can use the built-in express functionality like this:
`app.use(express.json());` "